tests/test_logger.py

```diff
@@ -325,6 +326,16 @@ def test_exception_handling(self):
                               {"color": "blue"},
                               ])
 
+    def test_overwriting_exception_handling(self):
+        factory = logging.getLogRecordFactory()
+        try:
+            with factory.add_context(color="blue"):
+                with factory.add_context(color="red"):
+                    raise RuntimeError("Something failed")
+        except RuntimeError as e:
+            # Preserved original context while escaping multiple managers?
```

Comment:
Do you mean to leave the "?" mark here?

Reply:
Sorry for the ambiguous grammar. Will fix.